pendulum_msgs/msg/JointState.msg

```diff
@@ -0,0 +1,3 @@
+float64 position
```

Comment:
I've made an issue for this idea, let's move the discussion there: ros2/ros2/issues/95